proteus/mprans/RANS2P.h

```diff
@@ -5240,9 +5103,9 @@ namespace proteus
 
                 if (NONCONSERVATIVE_FORM)
                   {
-                    bc_mom_w_adv_ext[0] -= MOVING_DOMAIN*bc_dmom_w_acc_w_ext*z_ext*xt_ext;
-                    bc_mom_w_adv_ext[1] -= MOVING_DOMAIN*bc_dmom_w_acc_w_ext*z_ext*yt_ext;
-                    bc_mom_w_adv_ext[2] -= MOVING_DOMAIN*bc_dmom_w_acc_w_ext*z_ext*zt_ext;
+                    bc_mom_w_adv_ext[0] -= MOVING_DOMAIN*bc_dmom_w_acc_w_ext*w_ext*xt_ext;
```

Comment:
@alistairbntl this needed to be the z component of velocity (w_ext), not the z coordinate.

Reply:
Thanks for pointing this out.  These only apply for the moving-mesh problems correct?  I need to familiarize myself a bit more with the methods for the moving-mesh.

Reply:
Yes, exacyly

Reply:
Yes, exacyly